charts/kpack/templates.yaml.tpl

```diff
@@ -0,0 +1 @@
+kpack: https://storage.googleapis.com/projectriff/internal/kpack/kpack-0.0.5-snapshot-5a4e635d.yaml
```

Comment:
should this be getting version from bucket?

Reply:
we'll switch to kpack releases as soon as they tag the next once. I think it's ok if we're a bit slower and deliberate with updating external components, especially when they are 0.0.x.